pkg/chartutil/capabilities.go

```diff
@@ -16,89 +16,67 @@ limitations under the License.
 package chartutil
 
 import (
-	"encoding/json"
-	"fmt"
-	"runtime"
-	"sort"
-
-	"k8s.io/apimachinery/pkg/version"
-
 	"k8s.io/client-go/kubernetes/scheme"
 )
 
 var (
 	// DefaultVersionSet is the default version set, which includes only Core V1 ("v1").
 	DefaultVersionSet = allKnownVersions()
 
-	// DefaultKubeVersion is the default kubernetes version
-	DefaultKubeVersion = &version.Info{
-		Major:      "1",
-		Minor:      "9",
-		GitVersion: "v1.9.0",
-		GoVersion:  runtime.Version(),
-		Compiler:   runtime.Compiler,
-		Platform:   fmt.Sprintf("%s/%s", runtime.GOOS, runtime.GOARCH),
-	}
-
 	// DefaultCapabilities is the default set of capabilities.
 	DefaultCapabilities = &Capabilities{
+		KubeVersion: KubeVersion{
+			Version: "v1.14.0",
+			Major:   "1",
+			Minor:   "14",
+		},
 		APIVersions: DefaultVersionSet,
-		KubeVersion: DefaultKubeVersion,
 	}
 )
 
-// Capabilities describes the capabilities of the Kubernetes cluster that Tiller is attached to.
+// Capabilities describes the capabilities of the Kubernetes cluster.
 type Capabilities struct {
-	// List of all supported API versions
+	// KubeVersion is the Kubernetes version.
+	KubeVersion KubeVersion
+	// APIversions are supported Kubernetes API versions.
 	APIVersions VersionSet
-	// KubeVerison is the Kubernetes version
-	KubeVersion *version.Info
 }
 
-// VersionSet is a set of Kubernetes API versions.
-type VersionSet map[string]struct{}
-
-// NewVersionSet creates a new version set from a list of strings.
-func NewVersionSet(apiVersions ...string) VersionSet {
-	vs := make(VersionSet)
-	for _, v := range apiVersions {
-		vs[v] = struct{}{}
-	}
-	return vs
+// KubeVersion is the Kubernetes version.
+type KubeVersion struct {
+	Version string // Kubernetes version
+	Major   string // Kubernetes major version
+	Minor   string // Kubernetes minor version
 }
 
+// String implements fmt.Stringer
+func (kv *KubeVersion) String() string { return kv.Version }
+
+// GitVersion returns the Kubernetes version string.
+//
+// Deprecated: use KubeVersion.Version.
```

Comment:
`KubeVersion.Version` isn't documented in the `builtin_objects.md` file. Can that be added alongside `Major` and `Minor`?